index.html

```diff
@@ -0,0 +1,122 @@
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+            </ul>
+
+        </nav>
+        <nav class="main-nav">
+            <a class="logo" href="index.html">Adonis</a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><button type="button">Campaign</button></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="home">
+        <div class="home-page">
+            <div class="menu-button">
+                <i></i>
+                <i></i>
+                <i></i>
+            </div>
+            <div>
+                <h2>"Hello! Sharing World!"</h2>
+                <h1>Peronal Development Seminar<br>2022</h1>
+            </div>
+            <p class="brief-description-home"> Lorem ipsum consectetur adipisicing elit dolor sit amet, consectetur
+                adipisicing consectetur adipisicing elit
+                elit. Iusto vitae illo cumque dolor fugiat quae aliquid.</p>
+            <div class="date-location">
+                <h2>2022.04.16(SAT) ~ 17(SUN)</h2>
+                <p>@ Nation Museum of Korea, Art Center nabi and more</p>
+            </div>
+        </div>
+    </section>
+    <section id="main">
+        <h1 class="section-headline">Main Program</h1>
+        <div class="main-container">
+            <div class="program">
+                <i class="icon-lecture"></i>
+                <h3>Lecture</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="icon-exhibition"></i>
+                <h3>Exhibition</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="icon-forum"></i>
+                <h3>Forum</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="icon-workshop"></i>
+                <h3>Workshop</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="icon-ignite"></i>
+                <h3>Lecture</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+        </div>
+        <button type="button">Join Lorem ipsum dolor</button>
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+    </section>

```

Comment:
Good job working on the project so far, I've only found one issue in the mobile view on the Home page.
- [ ] Kindly remove the excess white space that exists on mobile view on the Home page.
![image](https://user-images.githubusercontent.com/62290545/163421812-bc65b8fe-9e75-47fa-bb4e-aa641133eff8.png)

- [Hint] You can see from the following image below that this section is taking more width than it's supposed to. You can fix this easily by reducing the spacing between elements and making sure that it looks good on width `375px`, which is the mobile view's width. Good luck.

![image](https://user-images.githubusercontent.com/62290545/163421758-8a632a9c-1196-4008-8038-6f36394f65dd.png)